### PR TITLE
Removed deprecated stuff

### PR DESCRIPTION
**Pull request by core23**
## Subject
The will removed all `NEXT_MAJOR` fragments and deprecated methods/classes.

Please check this carefully @sonata-project/contributors.

I am targeting this branch, because this is a cleanup for the next major.

### CODE REVIEW

File: src/Block/Service/BlockServiceInterface.php [hunk 1]
```diff
@@ -13,9 +13,45 @@
...
+
 /**
  * @author Christian Gripp <mail@core23.de>
  */
-interface BlockServiceInterface extends \Sonata\BlockBundle\Block\BlockServiceInterface
+interface BlockServiceInterface
 {
+    /**
+     * @param BlockContextInterface $blockContext
+     * @param Response              $response
```
Comment (greg0ire) [inline src/Block/Service/BlockServiceInterface.php]: No need for this part

File: src/Block/Service/BlockServiceInterface.php [hunk 2]
```diff
@@ -13,9 +13,45 @@
...
+    public function execute(BlockContextInterface $blockContext, Response $response = null);
+
+    /**
+     * @return string
+     */
+    public function getName();
+
+    /**
+     * @param BlockInterface $block
+     */
```
Comment (greg0ire) [inline src/Block/Service/BlockServiceInterface.php]: or this docblock

File: src/Block/Service/BlockServiceInterface.php [hunk 3]
```diff
@@ -13,9 +13,45 @@
...
+    public function load(BlockInterface $block);
+
+    /**
+     * @param BlockInterface $block
+     *
```
Comment (greg0ire) [inline src/Block/Service/BlockServiceInterface.php]: Or this docblock

File: src/Block/Service/BlockServiceInterface.php [hunk 4]
```diff
@@ -13,9 +13,45 @@
...
+     * @return array
+     */
+    public function getCacheKeys(BlockInterface $block);
+
+    /**
+     * Define the default options for the block.
+     *
+     * @param OptionsResolver $resolver
```
Comment (greg0ire) [inline src/Block/Service/BlockServiceInterface.php]: and so on

File: src/Block/Service/BlockServiceInterface.php (hunk 1 above)
Comment (core23) [reply src/Block/Service/BlockServiceInterface.php]: Not part of this PR. I want to create atomic PRs with different topic: Remove stuff, Finalize classes/methods, cleanup, ...

File: src/Command/DebugBlocksCommand.php [hunk 5]
```diff
@@ -25,9 +25,8 @@ class DebugBlocksCommand extends BaseCommand
      */
     public function configure(): void
     {
-        // NEXT_MAJOR: Switch name and alias
-        $this->setAliases(['debug:sonata:block']);
-        $this->setName('sonata:block:debug');
+        $this->setAliases(['sonata:block:debug']);
+        $this->setName('debug:sonata:block');
```
Comment (OskarStark) [inline src/Command/DebugBlocksCommand.php]: Tbh I don’t think we need this.

A name with sonata:block... should be enough

File: tests/Block/BlockRendererTest.php [hunk 6]
```diff
@@ -1,5 +1,7 @@
 <?php
 
+/** @noinspection ALL */
```
Comment (kunicmarko20) [inline tests/Block/BlockRendererTest.php]: do we use any inspection tool?

Comment (core23) [reply tests/Block/BlockRendererTest.php]: Ooops

**Comment by core23** [comment]
ping @OskarStark

**Pull request merged**